recipe/meta.yaml

```diff
@@ -1,38 +1,35 @@
 {% set name = "circus" %}
-{% set version = "0.14.0" %}
-{% set sha256 = "593d26fbd43e8ec8088178101696bd776fa881d3f7d9eb5ab96cafda913c7dac" %}
+{% set version = "0.15.0" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: https://github.com/circus-tent/{{ name }}/archive/{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d1603cf4c4f620ce6593d3d2a67fad25bf0242183ea24110d8bb1c8079c55d1b
 
 build:
-  number: 1001
+  number: 0
   entry_points:
     - circusd = circus.circusd:main
     - circusd-stats = circus.stats:main
     - circusctl = circus.circusctl:main
     - circus-top = circus.stats.client:main
     - circus-plugin = circus.plugins:main
   script: "{{ PYTHON }} -m pip install . --no-deps -vv"
-  skip: True  # [win]
-
+  skip: True  # [win or py>35]
```

Comment:
@ltalirz do you agree with this python version constraint? This is what I understand from the circus code and docs.

Reply:
hm... the [setup.py](https://github.com/circus-tent/circus/blob/b5b485dec09f3f1c8a45f0e8781b771018bd37a3/setup.py#L32-L39) says something else.
What makes you think that only python up to 3.4 is supported?

Reply:
actually, I'm looking at [this setup.py](https://github.com/circus-tent/circus/blob/0.15.0/setup.py) since it is the one tagged 0.15.0, and I'm inferring from [this](https://github.com/circus-tent/circus/blob/0.15.0/docs/source/changelog.rst#013---2016-01-27) changelog entry.

Reply:
Sorry, I should have done the same.
Your reasoning makes absolute sense - we are using circus 0.15 on python 3.6 without issues so far, though, and it would be great to be able to do this in conda as well.

Do you want me to open another issue concerning this?
Or do you happen to know a circus developer we could ping here?

Reply:
Well, I can enable python 3.6 here, I see no problem.